backend/src/abilities/tests/ability-fixture.ts

```diff
@@ -71,6 +88,48 @@ export function createAbilityFixture(): AbilityFixture {
         .withUpdatedAt(currentDate);
 
       return builder.build();
+    },
+    
+    // Result and error handling
+    getResult(): any {
+      return result;
+    },
+    
+    setResult(newResult: any): void {
```

Comment:
Code Review: The parameter type 'any' is too broad. Consider using a more specific type or interface to better define the structure of the result being set.